swagger.json

```diff
@@ -1121,6 +1121,116 @@
         ]
       }
     },
+    "/{vocid}/new": {
+      "get": {
+        "summary": "New concepts in the vocabulary",
+        "parameters": [
+          {
+            "name": "vocid",
+            "in": "path",
+            "description": "a Skosmos vocabulary identifier e.g. \"stw\" or \"yso\"",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "lang",
+            "in": "query",
+            "description": "label language, e.g. \"en\" or \"fi\"",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "description": "offset of the starting index",
+            "required": false,
+            "type": "number"
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "description": "maximum number of concepts to return",
+            "required": false,
+            "type": "number"
+          }
+        ],
+        "produces": [
+          "application/ld+json"
+        ],
+        "responses": {
+          "200": {
+            "description": "list of most recently created concepts of the vocabulary",
+            "schema": {
+              "$ref": "#/definitions/changedConceptsResult"
+            }
+          },
+          "304": {
```

Comment:
I see no indication that 304 responses would be supported by the `new` method? If they are not supported, then this response code should be omitted from the Swagger spec.